app/tools/risk-survey/[resultId]/page.tsx

```diff
@@ -28,10 +59,12 @@ const RiskSurveyResults = async (props: Props) => {
   const means = await readPerceivedInvestmentRiskAverages();
 
   return (
-    <Flex flexDir={"column"} gap={4}>
-      <Flex flexDir={"column"}>
-        <Heading>Risk Survery Results</Heading>
-        <Text>
+    <Flex flexDir={"column"} gap={12}>
```

Comment:
This component is HUGE, I would suggest breaking into different, smaller, compoenents. Like one for the accordian.

Reply:
Good point! I will do this before merging!